lib/classes/PluginManager.js

```diff
@@ -215,6 +220,27 @@ class PluginManager {
     );
   }
 
+  getLocalPluginsPathPatterns() {
+    return this.localPluginsPaths.map(({ resolvedPath, inputPath }) => {
+      const absoluteInputPath = path.resolve(this.serverless.serviceDir, inputPath);
+      if (
+        absoluteInputPath === resolvedPath ||
+        absoluteInputPath + path.extname(resolvedPath) === resolvedPath
+      ) {
+        return path.relative(this.serverless.serviceDir, resolvedPath);
+      }
+
+      if (resolvedPath.startsWith(absoluteInputPath + path.sep)) {
+        return path.relative(
+          this.serverless.serviceDir,
+          path.join(path.dirname(resolvedPath), '/**')
+        );
+      }
+
+      return '';
```

Comment:
I think we should return `null` and add filter to not return this value in collection

Reply:
Gotcha.